apis/upbound/mcp-connector/composition.yaml

```diff
@@ -0,0 +1,161 @@
+              string:
+                fmt: "%s-mcp-connector"
+    - name: mcp-token
+      base:
+        apiVersion: iam.upbound.io/v1alpha1
+        kind: Token
+        spec:
+          forProvider:
+            owner:
+              type: robots
+              idSelector:
+                matchControllerRef: true
+          writeConnectionSecretToRef:
+            namespace: upbound-system
+      patches:
+        - fromFieldPath: spec.parameters.deletionPolicy
+          toFieldPath: spec.deletionPolicy
+        - fromFieldPath: metadata.labels[crossplane.io/claim-name]
+          toFieldPath: spec.forProvider.name
+          transforms:
+            - type: string
+              string:
+                fmt: "%s-mcp-connector-access-token"
+        - fromFieldPath: metadata.labels[crossplane.io/claim-name]
+          toFieldPath: spec.writeConnectionSecretToRef.name
+          transforms:
+            - type: string
+              string:
+                fmt: "%s-mcp-connector"
+    - name: mcp-team
+      base:
+        apiVersion: iam.upbound.io/v1alpha1
+        kind: Team
+      patches:
+        - fromFieldPath: spec.parameters.deletionPolicy
+          toFieldPath: spec.deletionPolicy
+        - fromFieldPath: spec.parameters.organizationName
+          toFieldPath: spec.forProvider.organizationName
+        - fromFieldPath: metadata.labels[crossplane.io/claim-name]
+          toFieldPath: spec.forProvider.name
+          transforms:
+            - type: string
+              string:
+                fmt: "%s-mcp-connector"
+    - name: mcp-team-membership
+      base:
+        apiVersion: iam.upbound.io/v1alpha1
+        kind: RobotTeamMembership
+        spec:
+          forProvider:
+            robotIdSelector:
+              matchControllerRef: true
+            teamIdSelector:
+              matchControllerRef: true
+      patches:
+        - fromFieldPath: spec.parameters.deletionPolicy
+          toFieldPath: spec.deletionPolicy
+    - name: mcp-controlplane-permission
+      base:
+        apiVersion: mcp.upbound.io/v1alpha1
+        kind: ControlPlanePermission
+        spec:
+          forProvider:
+            teamIdSelector:
+              matchControllerRef: true
+            permission: owner
+      patches:
+        - fromFieldPath: spec.parameters.deletionPolicy
+          toFieldPath: spec.deletionPolicy
+        - fromFieldPath: spec.parameters.organizationName
+          toFieldPath: spec.forProvider.organizationName
+        - fromFieldPath: spec.parameters.controlPlaneName
+          toFieldPath: spec.forProvider.controlPlaneName
+    - name: mcp-connector
+      base:
+        apiVersion: helm.crossplane.io/v1beta1
+        kind: Release
+        spec:
+          rollbackLimit: 3
+          forProvider:
+            namespace: kube-system
+            chart:
+              name: mcp-connector
+              url: https://charts.upbound.io/beta/mcp-connector-0.3.2.tgz
+              version: 0.3.2
```

Comment:
are we intentionally hardcoding the version?

Reply:
We overriding with patch will remove it